test/grisp_connect_api_SUITE.erl

```diff
@@ -61,20 +58,49 @@ auto_connect_test(_) ->
     ?assertMatch(ok, wait_connection()).
 
 ping_test(_) ->
-    ?assertMatch({ok, <<"pang">>}, grisp_connect:ping()).
+    {Pid, MRef} = ?asyncAssertEqual({ok, <<"pong">>}, grisp_connect, ping, []),
+    Id = ?receiveJsonRpcRequest(<<"post">>, #{type := <<"ping">>}),
+    send_jsonrpc_result(<<"pong">>, Id),
+    ?asyncWait(Pid, MRef).
```

Comment:
Would that make more sense to do the assert in this than in `?asyncAssertEqual` ?
```
OpaqueThing = ?asyncCall(grisp_connect:ping()), % If it is a macro it could be done like this, but maybe it is too confusing...
...
..
_Result = ?assertAsyncCall(OpaqueThing, {ok, <<"pang">>}),
```